mlir/test/lib/Dialect/Test/TestDialect.cpp

```diff
@@ -228,6 +332,7 @@ void TestDialect::initialize() {
       >();
   registerOpsSyntax();
   addOperations<ManualCppOpWithFold>();
+  registerTestDialectOperations(this);
```

Comment:
This the actual "magic" right? You're calling into the generated registration method which dispatch to the shards?

What does the:
```
  addOperations<
#define GET_OP_LIST
#include "TestOps.cpp.inc"
       >();
```

still do above?

Reply:
As you noted, this isn't needed. I think this got lost in the merge resolution. I'll delete it.